### PR TITLE
Bugs on engagement banner

### PR DESCRIPTION
**Pull request by zeftilldeath**
Thank you @walaura  for reporting. 

# Before
screen shot 2019-02-13 at 15 47 05

# After
screen shot 2019-02-13 at 15 47 19

# Before
screen shot 2019-02-13 at 15 47 36

# After
screen shot 2019-02-13 at 15 47 52

### CODE REVIEW

**Comment by joelochlann** [comment]
thanks @walaura and @zeftilldeath!

File: static/src/stylesheets/module/site-messages/_engagement-banner.scss [hunk 1]
```diff
@@ -94,28 +95,33 @@
     vertical-align: middle;
 
     height: 36px;
-    padding-right: 0;
+    padding: 0 ($gs-gutter / 3) 0 18px;
```
Comment (walaura) [inline static/src/stylesheets/module/site-messages/_engagement-banner.scss]: where is this `18` coming from?

**Review by walaura — APPROVED** [review]
😻

**Comment by walaura** [comment]
Q: does this fix the roundel alignment on the non stacked cookie banner?

File: static/src/stylesheets/module/site-messages/_engagement-banner.scss (hunk 1 above)
Comment (zeftilldeath) [reply static/src/stylesheets/module/site-messages/_engagement-banner.scss]: My mind 🐡

**Pull request merged**